scripts/remove

```diff
@@ -49,7 +50,7 @@ ynh_remove_app_dependencies
 #=================================================
 # REMOVE APP MAIN DIR
 #=================================================
-ynh_script_progression --message="Removing Wetty main directory..." --weight=1
+ynh_script_progression --message="Removing app main directory..." --weight=1
```

Comment:
$app maybe ?

Reply:
It could but trying to stick to https://github.com/YunoHost/example_ynh/blob/e3878cde95552282313ffe60cd52f539f3ff93c8/scripts/remove#L65